src/components/Footer/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import cx from 'classnames/bind'
+import style from './style.scss'
+
+const classNames = cx.bind(style)
+
+const Footer = () => (
+  <div className={classNames('Footer')}>
```

Comment:
If it is only static class names, how about use the string directly, like `className="Footer"`?